Formula/pants.rb

```diff
@@ -0,0 +1,36 @@
+class Pants < Formula
+  desc "Fast, scalable, user-friendly build system for codebases of all sizes"
+  homepage "https://pantsbuild.org"
+  url "https://github.com/pantsbuild/scie-pants/archive/refs/tags/v0.2.0.tar.gz"
+  sha256 "bebe326350ee7477bc1ac8c9c79222e987e3ee01500b4e16a526134529eb9866"
+  license "Apache-2.0"
+  head "https://github.com/pantsbuild/scie-pants.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on "rustup-init" => :build
```

Comment:
It should build with the default rust in Homebrew

Reply:
It does, except for a weird use of `rustup` in the projects rust package tool:

https://github.com/pantsbuild/scie-pants/blob/f2bf258d6d5cac0913f2ca94064941ad7d92fccd/package/src/main.rs#L520

As the comment above suggests, this will likely go away, and then we can revert this to just depend on `rust`.

Reply:
Is this regarded a blocker in this case?

I was considering creating a stub `rustup` script to just make that line a no-op during packaging. Is that a preferred approach? (or, we'll hold this until fixed in the scie-pants repo)

Reply:
It would definitely need to be fixed upstream before it can be merged here

Reply:
OK, thanks.

I notice there's a failing test also for the GH project notability check, will that be circumventable? otherwise this'll be a no-go for now (and some foreseeable future, in which case I guess setting up a custom tap would be our next option)

Reply:
How about re-using the built releases from the upstream project for installation, rather than re-building them again here?

https://github.com/pantsbuild/scie-pants/releases/tag/v0.2.0

Would that be acceptable?

Reply:
`homebrew-core` builds from source from all formula. We would not be able to accept a formula that reuses already built binaries.

Reply:
OK, thanks.